config/configure.in

```
@@ -563,6 +563,40 @@ else
   AC_MSG_RESULT([found $NUMS_INCLUDE])
 fi
 
+CURL=yes
+{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking ocaml curl library" >&5
+AC_MSG_CHECKING([ocaml curl library]); }
```

`AC_MSG_CHECKING()` was meant to substitute the whole two lines, not only the second `printf`. Like:

```suggestion
AC_MSG_CHECKING([ocaml curl library])
```